rpm/spec.in

```diff
@@ -83,5 +83,10 @@ mkdir -p $RPM_BUILD_ROOT/
 %{_includedir}/libdrizzle-5.1/visibility.h
 
 %changelog
+* Fri September 21 2016 Andreas Bok Andersen <andreas.bok@sociomantic.com> - v5.3.1 
+- Bump versions for patch release v5.3.1
```

Comment:
This last line is kind of too redundant and not really saying what changed in this version :)